toggl/api/models.py

```diff
@@ -454,6 +454,23 @@ class Task(PremiumEntity):
     """
 
 
+class TagSet(base.WorkspaceTogglSet):
+    pass
+
+
+class Tag(WorkspacedEntity):
+    """
+    Tag entity
+    """
+
+    name = fields.StringField(required=True)
+    """
+    Name of tag (Required)
+    """
+
+    objects = TagSet()
```

Comment:
You don't have to implement your own Set unless you need to implement some custom behavior which is not the case here. Every Entity get automatically assigned `WorkspaceTogglSet` as you can see here: https://github.com/AuHau/toggl-cli/blob/master/toggl/api/base.py#L377

So please remove the TagSet.

Reply:
Ah, so that's how that works. First use-case of metaclasses I've seen in person :)

Reply:
Hehe, yeah metaclasses are cool, but you don't usually work with them that much unless you are making some sort of framework or something :-)